utils.py

```diff
@@ -208,52 +209,54 @@ def getDialog():
-    try:
-        with open('/etc/os-release', 'r') as f:
-            os_release_content = f.read()
-        match = re.search(r'^ID=(\S+).*?VERSION_ID=(\S+)', os_release_content, re.MULTILINE)
-        if match:
-            config.OS_NAME = match.group(1)
-            config.OS_RELEASE = match.group(2)
-            return config.OS_NAME, config.OS_RELEASE
-    except FileNotFoundError:
-        pass
-
-    # Try using lsb_release command
-    try:
-        config.OS_NAME = platform.linux_distribution()[0]
-        config.OS_RELEASE = platform.linux_distribution()[1]
-        return config.OS_NAME, config.OS_RELEASE
-    except AttributeError:
-        pass
-
-    # Try reading /etc/lsb-release
-    try:
-        with open('/etc/lsb-release', 'r') as f:
-            lsb_release_content = f.read()
-        match = re.search(r'^DISTRIB_ID=(\S+).*?DISTRIB_RELEASE=(\S+)', lsb_release_content, re.MULTILINE)
-        if match:
-            config.OS_NAME = match.group(1)
-            config.OS_RELEASE = match.group(2)
-            return config.OS_NAME, config.OS_RELEASE
-    except FileNotFoundError:
-        pass
+    # TODO: Remove if we can verify these are no longer needed commented code.
 
-    # Try reading /etc/debian_version
-    try:
-        with open('/etc/debian_version', 'r') as f:
-            config.OS_NAME = 'Debian'
-            config.OS_RELEASE = f.read().strip()
-            return config.OS_NAME, config.OS_RELEASE
-    except FileNotFoundError:
-        pass
+    # Try reading /etc/os-release
+    #try:
+    #    with open('/etc/os-release', 'r') as f:
+    #        os_release_content = f.read()
+    #    match = re.search(r'^ID=(\S+).*?VERSION_ID=(\S+)', os_release_content, re.MULTILINE)
+    #    if match:
+    #        config.OS_NAME = match.group(1)
+    #        config.OS_RELEASE = match.group(2)
+    #        return config.OS_NAME, config.OS_RELEASE
+    #except FileNotFoundError:
+    #    pass
+
+    ## Try using lsb_release command
+    #try:
+    #    config.OS_NAME = platform.linux_distribution()[0]
+    #    config.OS_RELEASE = platform.linux_distribution()[1]
+    #    return config.OS_NAME, config.OS_RELEASE
+    #except AttributeError:
+    #    pass
+
+    ## Try reading /etc/lsb-release
+    #try:
+    #    with open('/etc/lsb-release', 'r') as f:
+    #        lsb_release_content = f.read()
+    #    match = re.search(r'^DISTRIB_ID=(\S+).*?DISTRIB_RELEASE=(\S+)', lsb_release_content, re.MULTILINE)
+    #    if match:
+    #        config.OS_NAME = match.group(1)
+    #        config.OS_RELEASE = match.group(2)
+    #        return config.OS_NAME, config.OS_RELEASE
+    #except FileNotFoundError:
+    #    pass
+
+    ## Try reading /etc/debian_version
+    #try:
+    #    with open('/etc/debian_version', 'r') as f:
+    #        config.OS_NAME = 'Debian'
+    #        config.OS_RELEASE = f.read().strip()
+    #        return config.OS_NAME, config.OS_RELEASE
+    #except FileNotFoundError:
+    #    pass
```

Comment:
Once we know that hte below `distro.*()` functions work, we can remove this code.